packages/core/src/interactions/network/heartbeat.ts

```diff
@@ -83,14 +79,12 @@ class Heartbeat implements AbstractInteraction {
         error(`heartbeat connection error ${err.name} while dialing ${counterparty.toB58String()} (initial)`, err)
       }
 
-      console.log(`struct`, struct)
-
       if (abort.signal.aborted) {
         return reject()
       }
 
       if (struct == null) {
-        const { id, multiaddrs } = await this.node.peerRouting.findPeer(counterparty)
```

Comment:
Was this needed to be removed?

Reply:
Yes, was only necessary for debug output